easybuild/easyconfigs/q/Qt/Qt-5.15.2-CrayGNU-20.11.eb

```diff
@@ -0,0 +1,26 @@
+# contributed by Luca Marsella (CSCS)
+name = 'Qt'
+version_major_minor = '5.15'
```

Comment:
```suggestion
```

Reply:
@ChristopherBignamini you don't need to define `version_major_minor`, it's a template defined by EasyBuild from the `version` parameter